### DISCUSSION TITLE
Steal 6328's wheel size characterization?

### DISCUSSION THREAD

**Discussion by PChild** [team401/2024-Robot-Code#136] [Ideas]
Can we steal [this wheel characterization](https://www.chiefdelphi.com/t/frc-6328-mechanical-advantage-2024-build-thread/442736/264#wheel-radius-characterization-7) really easily and test it? I'm still somewhat concerned about switching tread at event and losing odometry accuracy.

**Comment by jkleiber** [comment]
Created #137
